src/main/kotlin/study/step3/Car.kt

```diff
@@ -0,0 +1,16 @@
+package study.step3
+
+class Car(
+    private val movementConditionGenerator: MovementConditionGenerator
+) {
+    private var trace: String = ""
```

Comment:
코틀린은 getter/setter 접근자를 사용해 외부에 공개할 프로퍼티를 클래스 내부에서만 변경가능하도록 만들 수 있습니다🙂
```suggestion
    var trace: String = ""
        private set
```

Reply:
@saintbeller96
이 문법은 여러번 공부했었는데도, 익숙하지 않아서 실무에서 적용을 못하고 있었는데
이제 확실히 기억된것같네요 ㅋㅋ 👍  
감사합니다 😃